app/src/components/layout/LogoComponents/ConfigurableLogo.tsx

```diff
@@ -3,39 +3,46 @@ import styled from '@emotion/styled';
 
 import { loadConfig } from 'src/utils';
 import { Link } from 'react-router-dom';
-import { pxToRem } from 'src/styled-theme';
+import { breakpoints, pxToRem } from 'src/styled-theme';
 import { useAppWidth } from 'src/hooks';
+import { getIsFirstVisit, useAppSelector } from 'src/store';
 
 export const ConfigurableLogo: React.FC = () => {
-  const { logoUrl } = loadConfig();
+  const { logoUrl, logoSize } = loadConfig();
   const { isMobile } = useAppWidth();
+  const isFirstVisit = useAppSelector(getIsFirstVisit);
 
   return (
-    <Link to="/">
-      <ConfigurableLogoLink isMobile={isMobile}>
+    <ConfigurableLogoContainer
+      logoSize={logoSize}
+      isMobile={isMobile}
```

Comment:
This should be taken out completely I believe

Reply:
Removed!